app/src/main/java/com/compose/quotes_jetpack/MainActivity2.kt

```diff
@@ -105,12 +107,90 @@ class MainActivity2 : ComponentActivity() {
                                 fontWeight = FontWeight.Bold,
                                 fontSize = 15.sp
                             )
+
+                            Image(
+                                painter = painterResource(R.drawable.baseline_cancel_24),
+                                contentDescription = null,
+                                colorFilter = ColorFilter.tint(Color.Black),
+                                modifier = Modifier.padding(start = 280.dp)
+                                    .clickable {
+                                        opendialog.value = true
+                                        val data1 = it.favauthor
+                                        val data2 = it.favquote
+                                        CoroutineScope(Dispatchers.IO).launch {
+                                            try {
```

Comment:
This block of code is basically used to get the document reference id for the quote that is been stored on the Firestore database and using that document id we would be performing the delete operation.